### PR TITLE
Clean unused dependencies from BuildFiles

### PR DESCRIPTION
**Pull request by guitargeek**
#### PR description:

Another quick BuildFile cleaning PR in the style of many before (for example https://github.com/cms-sw/cmssw/pull/37915).

In many cases, the dependencies had to be moved to the appropriate BuildFile, for example from a plugins directory to the library directory or vice versa.

This PR cleans unnecessary includes from CMSSW BuildFiles that were added in the last months.

#### PR validation:

CMSSW compiles.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

No backport intended.

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by guitargeek** [comment]
Hi, sorry I opened the PR too early and realized that it didn't even compile yet locally... Now it does!

Sorry for wasting a bot test

**Comment by makortel** [comment]
@cmsbuild, please abort

Let's minimize the waste :)

**Comment by makortel** [comment]
@cmsbuild, please test

File: HeterogeneousCore/AlpakaServices/plugins/BuildFile.xml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 <!-- alpaka-based portable plugins -->
 <library file="alpaka/*.cc" name="HeterogeneousCoreAlpakaServicesPlugins">
   <use name="alpaka"/>
-  <use name="FWCore/MessageLogger"/>
-  <use name="FWCore/ParameterSet"/>
-  <use name="FWCore/PluginManager"/>
```
Comment (fwyzard) [inline HeterogeneousCore/AlpakaServices/plugins/BuildFile.xml:6]: Thanks for the clean up.

Do you mind if I cherry pick your commit for this file into a different branch, and make a separate PR ?
I'm trying to keep all changes to the Alpaka-related code in a dedicate branch, to ease backports _etc_.

Comment (guitargeek) [reply HeterogeneousCore/AlpakaServices/plugins/BuildFile.xml:6]: Yes that makes sense. I'll revert the change to this file in this PR then!

**Comment by guitargeek** [comment]
Force pushed to revert the changes to `HeterogeneousCore/AlpakaServices` as requested by @fwyzard

**Comment by ChrisMisan** [comment]
please test

File: DataFormats/PortableTestObjects/BuildFile.xml [hunk 2]
```diff
@@ -1,8 +1,6 @@
 <use name="rootcore"/>
 <use name="DataFormats/Common"/>
 <use name="DataFormats/Portable"/>
-<use name="FWCore/MessageLogger"/>
-<use name="FWCore/Utilities" source_only="1"/>
```
[the remark this answers is not in the document]
Comment (fwyzard) [reply DataFormats/PortableTestObjects/BuildFile.xml:5]: I missed it :-(
Yes, thanks for catching it. I'll move it to a separate branch/PR.

[the remark this answers is not in the document]
Comment (guitargeek) [reply DataFormats/PortableTestObjects/BuildFile.xml:5]: Okay, I also removed the change from this PR

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by srimanob** [comment]
+Upgrade

Technical PR.

**Comment by emanueleusai** [comment]
+1

File: DataFormats/PortableTestObjects/BuildFile.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fwyzard) [reply DataFormats/PortableTestObjects/BuildFile.xml:5]: can you actually drop the file from the commit ?
that should get rid of the `heterogeneous` signature.

**Comment by ChrisMisan** [comment]
+alca

- no significant differences in comparison
- PR changes are purely technical

**Comment by tvami** [comment]
+db
 * code compiles

**Comment by tvami** [comment]
@perrotta @qliphy @rappoccio @smuzaffar  I understand from this file 
https://github.com/cms-sw/cms-bot/blob/master/categories.py
there is nobody to sign for `analysis`, is the plan to just remove this signature altogether? Or we are waiting for somebody to accept that position? If yes, should ORP be attached to `analysis` in the meanwhile?

**Comment by perrotta** [comment]
@tvami we (ORP) are attached to analysis "in practice" since a while already...
As soon as @cms-sw/heterogeneous-l2 signs this can be considered as "fully signed"

**Comment by makortel** [comment]
> As soon as @cms-sw/heterogeneous-l2 and @cms-sw/l1-l2 sign, this can be considered as "fully signed"

@fwyzard asked earlier
> can you actually drop the file from the commit ?
> that should get rid of the `heterogeneous` signature.

in https://github.com/cms-sw/cmssw/pull/39160#discussion_r954595335.

**Comment by guitargeek** [comment]
My bad, I see I have re-added the line in the DataFormats section at the wrong place! It should be fine now, and the `heterogeneous` signature is not required anymore.

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by srimanob** [comment]
+Upgrade

Re-sign

**Comment by civanch** [comment]
+1

**Comment by ChrisMisan** [comment]
+alca

- re-signing
- differences in L1T not related to this PR

**Comment by mandrenguyen** [comment]
+reconstruction
re-sign

**Comment by tvami** [comment]
+db

**Comment by cecilecaillol** [comment]
+l1

**Comment by perrotta** [comment]
+1
- @cms-sw/dqm-l2 already approved [a few days ago](https://github.com/cms-sw/cmssw/pull/39160#issuecomment-1226807666) a version of this PR which is identical to this one for what the dqm-related files are concerned

**Comment by perrotta** [comment]
merge

**Comment by emanueleusai** [comment]
+1
